meeteval/viz/visualize.js

```diff
@@ -632,6 +633,23 @@ class CanvasPlot {
         //     redraw();
         // });
         // errorbar_mode_check.node().checked = settings.barplot.scaleExcludeCorrect;
+
+
+        // Audio
+        m.append("div").classed("divider", true);
+        m.append("div").classed("menu-section-label", true).text("Audio Server");
+        menuElement = m.append("div").classed("menu-element", true);
+        menuElement.append("div").classed("menu-label", true).text("Adress:")
+
+        menuElement
+            .append("input")
```

Comment:
Add `.classed("menu-control", true)` to get the correct alignment with the other menu entries

Reply:
Done